tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -83,8 +83,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     SdpSubarrayNodeFQDN = device_property(
-        dtype='str', default_value="mid_sdp/elt/subarray_1",
-        doc='FQDN of the SDP Subarray Node Tango Device Server.',
+        dtype='str', doc='FQDN of the SDP Subarray Node Tango Device Server.'
```

Comment:
Change the description. It should be 'FQDN of the SDP Subarray Tango Device Server.'

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd